src/services/DynamoDBService.ts

```diff
@@ -54,46 +52,53 @@ export class DynamoDBService {
             .promise();
     }
 
+    /**
+     * Scan the entire table and retrieve all data
+     * @returns Promise<PromiseResult<DocumentClient.ScanOutput, AWSError>>
+     */
+         public getActivities(filterParams: any): Promise<IActivity[]> {
```

Comment:
Why is this `any`? won't it always be `IActivityParams` ?

Reply:
updated to IActivityParams